test/conformance/ptp/ptp.go

```diff
@@ -516,6 +516,18 @@ var _ = Describe("[ptp]", func() {
 })
 
 func configurePTP() {
+	/*
+		The test takes significant amount of time checking
+		for the right interfaces to test
+		By setting the following variable environment for test purposes
+		The suite will check use them directly
+	*/
+
+	ptpGrandMasterNodeName := os.Getenv("PTP_CONFIG_MASTER_NODE")
+	masterInterfaceName := os.Getenv("PTP_CONFIG_MASTER_INTERFACE")
```

Comment:
what would be t default path if the environment is not set?

Reply:
an empty string.